runtime/ftplugin/man.vim

```diff
@@ -1,37 +1,26 @@
-" Vim filetype plugin file
-" Language:	man
-" Maintainer:	SungHyun Nam <goweol@gmail.com>
-
-if has('vim_starting') && &filetype !=# 'man'
-  finish
-endif
-
-" Only do this when not done yet for this buffer
 if exists('b:did_ftplugin')
   finish
 endif
 let b:did_ftplugin = 1
 
-" Ensure Vim is not recursively invoked (man-db does this)
-" when doing ctrl-[ on a man page reference.
-if exists('$MANPAGER')
-  let $MANPAGER = ''
-endif
-
-setlocal iskeyword+=\.,-,(,)
-
-setlocal buftype=nofile noswapfile
-setlocal nomodifiable readonly bufhidden=hide nobuflisted tabstop=8
+setlocal buftype=nofile
+setlocal noswapfile
+setlocal nofoldenable
+setlocal bufhidden=hide
+setlocal nobuflisted
+setlocal nomodified
+setlocal readonly
+setlocal nomodifiable
+setlocal noexpandtab
+setlocal tabstop=8
+setlocal softtabstop=8
+setlocal shiftwidth=8
+setlocal nolist
+setlocal foldcolumn=0
+setlocal colorcolumn=0
 
 if !exists("g:no_plugin_maps") && !exists("g:no_man_maps")
-  nnoremap <silent> <buffer> <C-]>    :call man#get_page(v:count, expand('<cword>'))<CR>
-  nnoremap <silent> <buffer> <C-T>    :call man#pop_page()<CR>
-  nnoremap <silent> <nowait><buffer>  q <C-W>c
-  if &keywordprg !=# ':Man'
-    nnoremap <silent> <buffer> K      :call man#get_page(v:count, expand('<cword>'))<CR>
-  endif
+  nnoremap <silent> <buffer> <C-]>    :call man#get_page(g:find_man_window, 'edit')<CR>
+  nnoremap <silent> <buffer> <C-t>    :call man#pop_tag_stack()<CR>
+  nnoremap <silent> <buffer> q :q<CR>
```

Comment:
This is not the same as `<C-W>c`. The old mapping was intentional. Can't change this.